TableController/TableController/Sources/TableController.swift

```diff
@@ -110,6 +118,33 @@ extension TableController {
 
 extension TableController: SectionControllerDelegate {
 
+    public func sectionControllerNeedsBeginUpdates(_ sectionController: SectionController) {
+        self.tableView.beginUpdates()
+    }
+    
+    public func sectionControllerNeedsEndUpdates(_ sectionController: SectionController) {
+        self.tableView.endUpdates()
+    }
+    
+    @available(iOS 11.0, *)
+    public func sectionController(_ sectionController: SectionController, needsBatchUpdates updates: () -> Void, completion: ((Bool) -> Void)?) {
+        self.tableView.performBatchUpdates(updates, completion: completion)
+    }
+    
+    public func sectionController(_ sectionController: SectionController, needsInsertRowsAt indices: Set<Int>, with animation: UITableViewRowAnimation) {
+        if let section = self.sectionControllers.index(where: { $0 === sectionController }) {
```

Comment:
The `self.sectionControllers.index(where: { $0 === sectionController })` bit was a hold over from when SectionController was just a protocol. We can go ahead and convert this usage to `index(of: sectionController)`

Reply:
Updated in ee3530a